include/triton/Tools/LinearLayout.h

```diff
@@ -238,6 +238,10 @@ namespace mlir::triton {
 // `a` as an M-bit integer, and each column Bi of the matrix B as an N-bit
 // integer.  Here's the same matrix-vector product written this way.
 //
+// Note:  B1 = [1, 0, 0, 0]^T
```

Comment:
Thanks for the PR.  The index changes are good, thanks.  However I don't think we need this section.  I think the endianness is clear because I give the binary and decimal interpretations.  And also the endianness is not important in the implementation.

If you remove this section I can merge the PR.

Reply:
(The endianness is not important because if we interpreted the numbers as "big-endian" instead, we'd get different decimal results but the same binary results.)

Reply:
Got it, thanks. I have removed.